spec/support/shared_contexts/with_user_logged_in.rb

```diff
@@ -1,5 +1,5 @@
 RSpec.shared_context 'with user logged in' do
-  let(:current_user) { build(:user) }
+  let(:current_user) { create(:user) }
 
```

Comment:
LOL Turned out: Just a few months ago, you updated from `build_stubbed` to `build`.  🤣